Configuration/ProcessModifiers/python/PixelCPEGenericForBricked_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is to run the Pixel Generic CPE algorithm specialized for bricked pixels in Phase-2 workflows
+PixelCPEGenericForBricked =  cms.Modifier()
```

Comment:
Convention is to start with lower case letter
```suggestion
pixelCPEGenericForBricked =  cms.Modifier()
```

Reply:
it was coded uppercase for consistency with 

https://github.com/cms-sw/cmssw/blob/master/Configuration/ProcessModifiers/python/PixelCPEGeneric_cff.py#L4